### PR TITLE
correct issues with inlineImages video element recording

### PR DESCRIPTION
**Pull request by Vadman97**
## Summary

* correct issue with `inlineImages` incorrectly inlining video size
* correct issue with video canvas recording breaking on video load (ios safari)

## How did you test this change?

Screenshot from 2024-06-27 14-47-39

https://app.highlight.io/1/sessions/ttIPIB1EhnR00ctKETRM2XZCXDha

## Are there any deployment considerations?

changeset

## Does this work require review from our design team?

no

### CODE REVIEW

File: frontend/src/index.tsx [hunk 1]
```diff
@@ -77,7 +76,7 @@ const options: HighlightOptions = {
 	debug: shouldDebugLog
 		? { clientInteractions: true, domRecording: true }
 		: undefined,
-	backendUrl: PUBLIC_GRAPH_URI,
+	// backendUrl: PUBLIC_GRAPH_URI,
```
Comment (SpennyNDaJets) [inline frontend/src/index.tsx:79]: Was this change intended?

**Review by SpennyNDaJets — APPROVED** [review]
LGTM!

File: frontend/src/index.tsx (hunk 1 above)
Comment (Vadman97) [reply frontend/src/index.tsx:79]: my bad, thanks for catching that!
